api/v1alpha1/tenantcontrolplane_status.go

```diff
@@ -244,3 +247,25 @@ type KubernetesIngressStatus struct {
 	// The namespace which the Ingress for the given cluster is deployed.
 	Namespace string `json:"namespace"`
 }
+
+type GatewayAccessPoint struct {
+	Type  *gatewayv1.AddressType `json:"type"`
+	Value string                 `json:"value"`
+	Port  int32                  `json:"port"`
+	URLs  []string               `json:"urls,omitempty"`
+}
+
+// +k8s:deepcopy-gen=false
+type RouteStatus = gatewayv1.RouteStatus
+
+// KubernetesGatewayStatus defines the status for the Tenant Control Plane Gateway in the management cluster.
+type KubernetesGatewayStatus struct {
+	// The TLSRoute status as resported by the gateway controllers.
+	RouteStatus `json:",inline"`
+
+	// Reference to the route created for this tenant.
+	RouteRef corev1.LocalObjectReference `json:"routeRef,omitempty"`
```

Comment:
Could we stick to the Kamaji way of referring to objects?

https://github.com/clastix/kamaji/blob/19d1fc39c3766d2bd0c0cfc30ce5f08a26e7d0a7/api/v1alpha1/tenantcontrolplane_status.go#L24-L26

Of course, instead of `secretName`, we could switch to `routeName`.